tools/metrics/histograms/metadata/cobalt/histograms.xml

```diff
@@ -55,6 +70,32 @@ Always run the pretty print utility on this file after editing:
   </summary>
 </histogram>
 
+<histogram name="Media.PipelineStatus.AudioVideo.{VideoCodec}.{Pipeline}"
+    enum="PipelineStatus" expires_after="never">
```

Comment:
You'll need an associated enums.xml entry for PipelineStatus. See enums docs at go/cobalt-telemetry#histograms

Reply:
done.